src/internal/serviceenv/config.go

```diff
@@ -45,7 +49,7 @@ type PachdFullConfiguration struct {
 type PachdSpecificConfiguration struct {
 	StorageConfiguration
 	NumShards                  uint64 `env:"NUM_SHARDS,default=32"`
-	StorageBackend             string `env:"STORAGE_BACKEND,default="`
+	StorageBackend             string `env:"STORAGE_BACKEND,default=LOCAL"`
```

Comment:
I don't think it's helpful to have this default.  It seems like when this is not set, it is better to error than silently switch to local.

Reply:
We can do that, this was just the existing behavior but I made it explicit.  I'll change this to be required.